src/redux/missions/missions.js

```diff
@@ -0,0 +1,35 @@
+// const GET_MISSIONS_FROM_API = 'SpaceX-Hub/missions/GET_MISSIONS_FROM_API';
+const GET_MISSIONS = 'spacex-hub/missions/GET_MISSIONS';
```

Comment:
- [ ] (OPTIONAL) It is good practice to remove all commented code to keep your work clean.